internal/git/gitsync_processor.go

```diff
@@ -114,27 +129,46 @@ func watchRepo(ctx context.Context, r *git.Repository, gitSync *v1alpha1.GitSync
 	val, ok := gitSync.Status.CommitStatus[repo.Name]
 	// Only create the resources for the first time if not created yet.
 	// Otherwise, monitoring with intervals.
+
+	repoAbsolutePath := localRepoPath + "/" + repo.Path
+	k := kustomize.NewKustomizeApp(repoAbsolutePath, repo.RepoUrl, "")
```

Comment:
nit: can this line go below where the kustomization logic is, or does it need to be over here? (seems more readable over there)

Reply:
I have put it here, so that I don't have to define it again for infinite for loop in line [here](https://github.com/numaproj-labs/numaplane/pull/86/files#diff-62540bd8eefb6e01f31efd127880279741d18ecc6845bc8f1f05a438db36a0b1R205)